src/Culling/ray.ts

```diff
@@ -245,6 +245,24 @@ export class Ray {
             return distance;
         }
     }
+    /**
+     * Calculate the intercept of a ray on a given axis
+     * @param axis to check 'x' | 'y' | 'z'
+     * @returns a vector containing the coordinates where 'axis' is equal to zero, or false if there is no intercept.
+     */
+    public intersectsAxis(axis) {
+          var t = this.origin[axis] / this.direction[axis]
+          if(t > 0) return false
+          const stepOrigin = function(t, axis) {

```

Comment:
for performance and GC reason we should not instantiate a function every time we call intersectsAxis